src/sharing/components/ShareByEmail.jsx

```diff
@@ -356,23 +275,13 @@ class ShareByEmail extends Component {
 }
 
 ShareByEmail.propTypes = {
-  contacts: PropTypes.array,
+  contacts: PropTypes.object.isRequired,
+  groups: PropTypes.object.isRequired,
```

Comment:
cozy-doctypes/groups / contacts should expose proptypes shape ?

Reply:
:+1: